### ISSUE TITLE
'invalid miner address or signature' in logs

### ISSUE THREAD

**Issue by cerblue** [textileio/bidbot#109]
I'm seeing a lot of `invalid miner address or signature` error messages in logs, along with `verifying miner address: calling full node state miner info: RPC client error: sendRequest failed: http status 503 Service Unavailable unmarshaling response: invalid character '<' looking for beginning of value`. My miner ([f010088](https://textileio.retool.com/embedded/public/46e74cd2-c47c-42ac-b542-189925795c41#provider=f010088)) was able to get some deals recently, so the error may be happening intermittently. I've tried changing Lotus API gateway URL in the bidbot config, but this error persists.

**Comment by jsign** [comment]
> invalid character '<' looking for the beginning of the value

Yeah, that's a signal that the GLIF API might have gone down temporarily.
I wouldn't worry much about it since that might happen once in a while (or longer if GLIF is having some particular longer downtime).
